src/NetAnalyzers/Core/Microsoft.NetFramework.Analyzers/MicrosoftNetFrameworkAnalyzersResources.resx

```diff
@@ -340,7 +340,7 @@
     <comment>CA3077</comment>
   </data>
   <data name="InsecureXsltScriptProcessingMessage" xml:space="preserve">
-    <value>Insecure XSLT script processing.</value>
+    <value>Insecure XSLT script processing</value>
     <comment>CA3076</comment>
   </data>
```

Comment:
This is passed only as a title, not a message. But I left the name as-is to minimize changes. Let me know if you want to rename this resource name.

Reply:
It’s fine to not change it in this PR